src/client.js

```diff
@@ -90,6 +92,29 @@ class CimpressTranslationsClient {
     return rp(options)
       .catch(requestCatch);
   }
+
+  async patchStructure(serviceId, structurePatches) {
+    let options = {
+      method: "PATCH",
+      url: this.buildUrl(API.v1ServicesIdStructure, { id: serviceId }),
+      body: structurePatches,
```

Comment:
We might want to validate that. At least to sanitise it a bit - otherwise we'll execute whatever the users sends us. We need to make sure it matches the specs of the RFC, not just the simplified syntax for adding a field.

Reply:
I added the JSON patch validation using the `fast-json-patch` library (which is also used to generate the patch in the `removeKeysFromStructure` function.